pkgs/applications/editors/vscode/extensions/default.nix

```diff
@@ -2903,6 +2903,22 @@ let
         };
       };
 
+      techtheawesome.rust-yew = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "rust-yew";
+          publisher = "techtheawesome";
+          version = "0.2.2";
+          sha256 = "sha256-t9DYY1fqW7M5F1pbIUtnnodxMzIzURew4RXT78djWMI=";
+        };
+        meta = {
+          description = "Provides some html language features for yew html macro.";
+          downloadPage = "https://marketplace.visualstudio.com/items?itemName=TechTheAwesome.rust-yew";
+          homepage = "https://github.com/TechTheAwesome/code-yew-server";
+          license = lib.licenses.gpl3;
```

Comment:
`lib.licenses.gpl3` is deprecated, please use `gpl3Plus` or `gpl3Only` as appropriate.